pkg/scalers/influxdb_scaler.go

```diff
@@ -47,7 +61,29 @@ func NewInfluxDBScaler(config *scalersconfig.ScalerConfig) (Scaler, error) {
 		return nil, fmt.Errorf("error parsing influxdb metadata: %w", err)
 	}
 
-	logger.Info("starting up influxdb client")
+	if meta.influxVersion == "3" {
+		logger.Info("starting up influxdb v3 client")
```

Comment:
this would be to verbose, let add `V(1)` here

Reply:
updated